package.json

```diff
@@ -111,68 +111,68 @@
     "npm-run-all": "^4.1.5",
     "postcss-loader": "^3.0.0",
     "rimraf": "^2.6.2",
-    "sass-loader": "^7.1.0",
+    "sass-loader": "^7.3.1",
     "script-loader": "^0.7.2",
     "style-loader": "^0.23.1",
-    "ts-jest": "^26.2.0",
+    "ts-jest": "^26.5.6",
     "ts-loader": "^5.3.1",
     "ts-node": "^7.0.1",
-    "tslint": "^5.12.0",
+    "tslint": "^5.20.1",
     "tslint-loader": "^3.6.0",
     "tslint-react": "^3.6.0",
-    "typescript": "^3.2.2",
+    "typescript": "^3.9.10",
     "url-loader": "^1.1.2",
-    "webpack": "^4.44.1",
+    "webpack": "^4.46.0",
     "webpack-cli": "^3.3.12",
-    "webpack-dev-server": "^3.11.0"
+    "webpack-dev-server": "^3.11.3"
   },
   "dependencies": {
     "@inlet/react-pixi": "^0.5.7",
-    "@material-ui/core": "^4.2.0",
+    "@material-ui/core": "^4.12.3",
     "@svgr/webpack": "^4.3.3",
-    "@types/d3": "^5.7.2",
-    "@types/d3-contour": "^1.3.0",
+    "@types/d3": "^5.16.4",
+    "@types/d3-contour": "^1.3.3",
     "@types/delaunator": "^3.0.0",
-    "@types/mathjs": "^6.0.5",
-    "@types/react-faux-dom": "^4.1.3",
-    "@types/react-leaflet": "^2.2.1",
-    "@types/recharts": "^1.1.16",
+    "@types/mathjs": "^6.0.12",
+    "@types/react-faux-dom": "^4.1.5",
+    "@types/react-leaflet": "^2.8.2",
+    "@types/recharts": "^1.8.23",
     "axios": "^0.21.4",
-    "color": "^3.1.0",
-    "d3": "^5.15.0",
+    "color": "^3.2.1",
+    "d3": "^5.16.0",
     "d3-contour": "^1.3.2",
     "deepmerge": "^4.2.2",
     "delaunator": "^4.0.1",
-    "iframe-phone": "^1.2.0",
+    "iframe-phone": "^1.3.1",
     "immutable": "^3.8.2",
-    "js-beautify": "^1.9.1",
+    "js-beautify": "^1.14.0",
     "js-interpreter": "^1.4.6",
     "leaflet": "^1.7.1",
     "leaflet-kmz": "^0.3.8",
     "less-loader": "^5.0.0",
     "lodash": "^4.17.21",
     "mathjs": "^7.6.0",
-    "mobx": "^5.8.0",
+    "mobx": "^5.15.7",
     "mobx-react": "^5.4.3",
-    "mobx-state-tree": "^3.9.0",
-    "pixi.js": "^4.8.7",
-    "prop-types": "^15.7.2",
-    "query-string": "^7.0.1",
-    "react": "^16.8.6",
-    "react-compound-slider": "^2.0.0",
-    "react-dat-gui": "^3.0.0",
-    "react-dom": "^16.8.6",
-    "react-draggable": "^4.4.3",
+    "mobx-state-tree": "^3.17.3",
+    "pixi.js": "^4.8.9",
+    "prop-types": "^15.8.1",
+    "query-string": "^7.1.1",
+    "react": "^17.0.2",
+    "react-compound-slider": "^3.3.1",
+    "react-dat-gui": "^4.1.1",
+    "react-dom": "^17.0.2",
+    "react-draggable": "^4.4.4",
     "react-faux-dom": "^4.5.0",
-    "react-leaflet": "^2.4.0",
+    "react-leaflet": "^2.8.0",
     "react-resize-observer": "^1.1.1",
     "react-syntax-highlighter": "^10.2.1",
-    "react-tabs": "^3.0.0",
-    "recharts": "^1.5.0",
+    "react-tabs": "^3.2.3",
+    "recharts": "^1.8.5",
     "screenfull": "^4.2.0",
-    "styled-components": "^5.1.1",
+    "styled-components": "^5.3.3",
     "transformation-matrix": "^1.15.3",
-    "uuid": "^3.3.2",
+    "uuid": "^3.4.0",
```

Comment:
It'd be worth trying to get to a newer version of `uuid` but check [Upgrading From uuid@3.x](https://www.npmjs.com/package/uuid#upgrading-from-uuid3x) for details.